txtorcon/torcontrolprotocol.py

```diff
@@ -103,6 +104,25 @@ def unlisten(self, cb):
 
     def got_update(self, data):
         # print self.name,"got_update:",data
+        if self.use_stem:
+            import stem.response
+            response_data = ""
+            # Add appropriate headers for parsing into stem.response.ControlMessage instance
+            if self.name == "CIRC":
+                response_data = "650 " + self.name + " " + data.replace('\n', ' ') + "\r\n"
+            elif self.name == "NEWCONSENSUS" or self.name == "NS":
+                lines = data.split('\n')
```

Comment:
Yes making changes in Stem will make things easier. But that would mean after the changes, a new version of stem is released that can be downloaded in Travis while running the tests. 
I'm currently looking at what the best way will be and will post the updates here.